# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1650999069 -7200
# Parent  cbacac19788ff9ff229d8883ffa29146988bddb0
Feature: Add more characters to Insert Characters/Symbols menu: Non-breaking hyphen \u2011, en-dash \u2013, em-dash \u2014

diff --git a/mail/components/compose/content/dialogs/EdInsertChars.js b/mail/components/compose/content/dialogs/EdInsertChars.js
--- a/mail/components/compose/content/dialogs/EdInsertChars.js
+++ b/mail/components/compose/content/dialogs/EdInsertChars.js
@@ -398,15 +398,15 @@ const lower = [
   "\u1e8b\u1e8d",
   // y
   "\u00fd\u00ff\u0177\u0233\u1e8f\u1e99\u1ef3\u1ef5\u1ef7\u1ef9",
   // z
   "\u017a\u017c\u017e\u0225\u1e91\u1e93\u1e95",
 ];
 
 const symbol =
-  "\u00a1\u00a2\u00a3\u00a4\u00a5\u20ac\u00a6\u00a7\u00a8\u00a9\u00aa\u00ab\u00ac\u00ae\u00af\u00b0\u00b1\u00b2\u00b3\u00b4\u00b5\u00b6\u00b7\u00b8\u00b9\u00ba\u00bb\u00bc\u00bd\u00be\u00bf\u00d7\u00f7";
+  "\u00a1\u00a2\u00a3\u00a4\u00a5\u20ac\u00a6\u00a7\u00a8\u00a9\u00aa\u00ab\u00ac\u00ae\u00af\u00b0\u00b1\u00b2\u00b3\u00b4\u00b5\u00b6\u00b7\u00b8\u00b9\u00ba\u00bb\u00bc\u00bd\u00be\u00bf\u00d7\u00f7\u2011\u2013\u2014";
 
 const otherupper =
   "\u00c6\u00d0\u00d8\u00de\u0132\u0152\u0186\u01c4\u01c5\u01c7\u01c8\u01ca\u01cb\u01F1\u01f2";
 
 const otherlower =
   "\u00e6\u00f0\u00f8\u00fe\u00df\u0133\u0153\u01c6\u01c9\u01cc\u01f3";
